cmake/Dependencies.cmake

```diff
@@ -84,17 +84,12 @@ endif()
 
 # ---[ On Android, Caffe2 uses cpufeatures library in the thread pool
 if (ANDROID)
-  # ---[ Check if cpufeatures was already imported
-  if (NOT TARGET cpufeatures)
-    add_library(cpufeatures STATIC
+  add_library(__caffe2_cpufeatures STATIC
```

Comment:
This will create problems if Caffe2 is included using `add_subdirectory` and the parent project also uses libcpufeatures (even if under a different target name) and links Caffe2 statically. If suggest to leave the old snippet, just remove `Caffe2_EXTERNAL_DEPENDENCIES` reference.